monitoring/tools/utils.py

```diff
@@ -6,11 +6,12 @@
 from odc.aws import s3_client, s3_fetch, s3_ls_dir
 
 
-def setup_logging() -> logging.Logger:
+def setup_logging(level: int = logging.INFO) -> logging.Logger:
```

Comment:
If the utils will be shared between **monitoring** and **data**, shouldn't we take it out of **monitoring**?

Reply:
Yes we should. I don't like how we have structured it.